src/helics/core/NetworkBrokerData.cpp

```diff
@@ -21,8 +21,8 @@ namespace helics
 static const ArgDescriptors extraArgs{
   {"interface"s, "the local interface to use for the receive ports"s},
   {"local_interface"s, "the local interface to use for the receive ports"s},
-  {"broker,b"s, "identifier for the broker"s},
-  {"broker_address"s, "location of the broker i.e network address"s},
+  {"broker"s, "identifier for the broker"s},
+  {"broker_address,b"s, "location of the broker i.e network address"s},
```

Comment:
Looking at the broker and broker_address options, it seems like the difference is that broker searches by name first, then tries to extract an address -- are there use cases where having the option to skip the broker search by name is required?

Reply:
Assuming I am understanding the question.  It is theoretically possible to have a broker in the local space but you want to connect with a broker on a different machine or process with the same name.    In which case you would potentially need both a name and address, which is why there are different options.    In most cases you need one or the other

Reply:
Okay, so that's a scenario where the broker address would need to be given to skip the local name search. I'm trying to find where the broker set using the option for a federate info structure is used -- I'm wondering if the search order used for that might be closer to the behavior of the broker option here rather than broker_address.

Reply:
for the federateInfo   everything in the broker option gets assigned to --broker when passed into the core.    It is equivalent to passing "--broker=..." in the coreinitstring

Reply:
so broker can be either the name or address

Reply:
Would it be more accurate to say that the --broker option in the broker is the broker's equivalent of the --broker option in federateinfo since both are looking for a name or address? In which case, changing the descriptions for the broker to say that --broker searches by name or address and that --broker_address sets the broker to search by name on (then -b could remain the short option for --broker).

Reply:
I updated the help line for the --broker argument and moved the -b back to it.